packages/libs/src/sdk/api/challenges/ChallengesApi.ts

```diff
@@ -327,7 +327,9 @@ export class ChallengesApi extends GeneratedChallengesApi {
       transactions.push(submitTransaction)
     }
     return await Promise.all(
-      transactions.map((t) => this.rewardManager.sendTransaction(t))
+      transactions.map((t) => {
+        return { transactionSignature: this.rewardManager.sendTransaction(t) }
```

Comment:
`this.rewardManager.sendTransaction(t)` should be awaited here

Reply:
oops good catch